.github/workflows/goreleaser.yml

```diff
@@ -37,3 +42,4 @@ jobs:
         args: release --rm-dist
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        docker_repository_owner: devopps
```

Comment:
This should be `cyclonedx`.

Reply:
Agree, missed to fix that before pushing the code